src/pages/SignPage/KakaoLogin/KakaoLogin.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+const KakaoLogin = () => {
+  const { Kakao } = window;
+  const [Login, setLogin] = useState(false);
+  const handlelogout = () => {
```

Comment:
- 한칸 띄워주시면 좋을 것 같습니다.
```suggestion
  const [Login, setLogin] = useState(false);
  
  const handlelogout = () => {
```